en/flight_controller/snapdragon_flight_advanced.md

```diff
@@ -36,105 +36,28 @@ adb shell
 
 ## Upgrade Snapdragon
 
-For this step the Flight_BSP zip file from Intrynsic is required. It can be obtained after registering using the board serial.
+For this step the files from the [Intrinsyc support](https://support.intrinsyc.com/projects/snapdragon-flight/files) website is required. Latest release that has been tested is `3.1.3.1`. It can be obtained after registering using the board serial.
```

Comment:
"is required" to "are required"

Reply:
"It can be obtained after registering using the board serial." I understand what "after registering" means, but what is "using the board serial"?